src/main/java/killjoy/processing/ProcessTasks.java

```diff
@@ -0,0 +1,200 @@
+package killjoy.processing;
+
+import killjoy.main.KillJoy;
+import killjoy.task.Task;
+import killjoy.main.UserInterface;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+/**
+ * Represents the ProcessTasks class of the KillJoy application.
+ * Contains methods to process user input.
+ */
+public class ProcessTasks {
+    private KillJoy kj;
+    private UserInterface ui;
+    private Parser parser = new Parser(ui);
+
+    public ProcessTasks(KillJoy kj, UserInterface ui) {
+        this.kj = kj;
+        this.ui = ui;
+    }
+
+    /**
+     * Processes the user input.
+     * @param input The user input.
+     */
+    public void processUserInput(String input) {
+        Task.TaskType taskType = parser.parseUserInput(input);
+
+        if (taskType == null) {
+            ui.displayUnknownCommandMessage();
+            return;
+        }
+
+        String[] inputSplitBySlash = input.split("/");
+
+        if (taskType != Task.TaskType.TODO && inputSplitBySlash.length == 1) {
+            ui.printLine();
+            ui.displayInvalidCommandFormatMessage();
+            ui.printLine();
+            return;
+        }
+
+        if (inputSplitBySlash[0].split(" ").length == 1) {
+            ui.printLine();
+            ui.displayInvalidCommandFormatMessage();
+            ui.printLine();
+            return;
+        }
+
+        ui.printLine();
+        switch (taskType) {
+        case TODO: {
```

Comment:
According to the coding standard website [here](https://se-education.org/guides/conventions/java/intermediate.html), there is no need for a pair of curly braces {} after each case. 😄

Reply:
I asked prof about it. He said that if it is not specified you can do it according your own liking but it has to be consistent. The thing is without braces, the variables declared can be used by other cases as well. To prevent it, I enclosed them in {}.